src/com/carolinarollergirls/scoreboard/core/impl/ClockImpl.java

```diff
@@ -55,7 +55,7 @@ else if (flag != Flag.INTERNAL) {
                     value = (((Long)value / 1000L) * 1000L) + (Long)last % 1000L;
                 }
             }
-            if ((flag == Flag.RESET && isCountDirectionDown()) || (Long)value > getMaximumTime() + 500) {
+            if ((flag == Flag.RESET && isCountDirectionDown()) || (Long)value > getMaximumTime() + 500 && !isCountDirectionDown()) {
```

Comment:
This would allow the user to set a time above max for countdown clocks.
Using `(!isCountDirectionDown() || flag != Flag.RECALCULATE)` should work. (Recalculate is only triggered when min or max are changed. Changing min can't trigger this condition, changing max is supposed to be excluded.)

Reply:
Done